CHANGELOG.md

```diff
@@ -2,6 +2,34 @@
 
 The release log for BoTorch.
 
+## [0.9.5] -- Dec 8, 2023
+
+#### New features
+
+Hypervolume Knowledge Gradient (HVKG):
+* Add `qHypervolumeKnowledgeGradient`, which seeks to maximize the difference in hypervolume of the hypervolume-maximizing set of a fixed size after conditioning the unknown observation(s) that would be received if X were evaluated (#1950).
+* Add initializer for one-shot HVKG (#1982).
+* Add tutorial on decoupled Multi-Objective Bayesian Optimization (MOBO) with HVKG (#2094).
+* Illustrate how to use Multi-Fidelity HVKG (MV-HVKG) (#2101).
```

Comment:
It feels like rather than having a line item for each PR, can we logically group the PRs together? 
```suggestion
* Add `qHypervolumeKnowledgeGradient`, which seeks to maximize the difference in hypervolume of the hypervolume-maximizing set of a fixed size after conditioning the unknown observation(s) that would be received if X were evaluated (#1950, #1982, #2101).
* Add tutorial on decoupled Multi-Objective Bayesian Optimization (MOBO) with HVKG (#2094).
```